src/marten/cli/manage/command/clearsessions.cr

```diff
@@ -0,0 +1,41 @@
+module Marten
+  module CLI
+    class Manage
+      module Command
+        class ClearSessions < Base
+          command_name :clearsessions
+          help "Clears all expired sessions."
+
+          @no_prompt : Bool = false
+
+          def setup
+            on_option("-y", "Do not show prompts to the user") { @no_prompt = true }
```

Comment:
Can we use the `no-input` instead of `-y` to be consistent with similar options in other commands (for example like in the [`collectassets`](https://github.com/martenframework/marten/blob/3e7365566e7c955d8c927d29ee0fbf74d5dc2a4c/src/marten/cli/manage/command/collect_assets.cr#L12) command)?

```suggestion
            on_option("no-input", "Do not show prompts to the user") { @no_input = true }
```